studio/schemas/documents/podcastEpisode.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Here is an example podcast as stored in egghead-rails:
+
+#<Podcast:0x00007f805f2e74a8
```

Comment:
We can delete this when we're done, just wanted it as a reference point for what values we may want to include in the migration.